controllers/user.server.controller.js

```diff
@@ -109,7 +109,7 @@ exports.userUpdateById = function(req, res) {
             "username": reqBody.username && reqBody.username.length > 2 ? reqBody.username : false,
             "displayName": reqBody.displayName && reqBody.displayName.length > 2 ? reqBody.displayName : false,
             "email": reqBody.email && emailValidator.validate(reqBody.email) ? reqBody.email : false,
-            "hashedPassword": reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
+            "password": reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
```

Comment:
```suggestion
            "plaintextPassword": reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
```
If we are going to use a more readable name then why not use plaintextPassword to more clearly define that it is not hashed yet